src/integrations/prefect-docker/prefect_docker/worker.py

```diff
@@ -389,8 +409,16 @@ def __init__(
             self.test_mode = bool(os.getenv("PREFECT_DOCKER_TEST_MODE", False))
         else:
             self.test_mode = test_mode
+        self.__tmp_dir = None
```

Comment:
I want to call out that the double underscore can produce unexpected issues accessing this attribute:

```python
class A:
    def __init__(self, x):
        self.__x = x
    def getx(self):
        return self.__x

a = A(x=4)
a.getx() # returns 4
a.__x # AttributeError
```

I don't think it matters because you're accessing it privately within methods but wanted to flag either way

Reply:
Yeah, access should only be done internally via the `_tmp_dir` property, which is there to lazily create a temporary directory only when needed.